### PR TITLE
Update main.go

### PR DESCRIPTION
**Pull request by heyztb**
Perhaps someone forgot to update this package when the API for go-rod changed. Simple fix.

### CODE REVIEW

**Review by ysmood — CHANGES_REQUESTED** [review]
Thank you. Nice catch. Please make sure to commit the change of `go mod tidy`. It will modify the `go.sum` file.

File: main.go [hunk 1]
```diff
@@ -8,17 +8,17 @@ import (
 
 // PageE creates a stealth page that can't be detected as bot.
 func PageE(b *rod.Browser) (*rod.Page, error) {
```
Comment (ysmood) [inline main.go:10]: Can you also update the name convention of the `PageE` and `Page`?

**Review by ysmood — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (heyztb) [reply main.go:10]: Yea sure, will commit shortly

**Comment by heyztb** [comment]
Should be all good to go - though I'm receiving different results from https://bot.sannysoft.com than what's included in the example file.

It seems there's more work to be done to get this library back to passing the checks offered by that site

**Comment by ysmood** [comment]
> Should be all good to go - though I'm receiving different results from https://bot.sannysoft.com than what's included in the example file.

The steps to fix:

1. run `go generate`
2. test the `example.go`
3. commit the auto-updated source files.

**Review by ysmood — APPROVED** [review]

**Pull request merged**